### PR TITLE
feat(components): [el-slider] slider add size prop

### PR DESCRIPTION
**Pull request by kooriookami**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by sxzz — APPROVED** [review]

File: docs/en-US/component/slider.md [hunk 1]
```diff
@@ -97,7 +106,8 @@ slider/show-marks
 | step                  | step size                                                                                                 | number          | —                      | 1       |
 | show-input            | whether to display an input box, works when `range` is false                                              | boolean         | —                      | false   |
 | show-input-controls   | whether to display control buttons when `show-input` is true                                              | boolean         | —                      | true    |
-| input-size            | size of the input box                                                                                     | string          | large / default /small | default |
+| size                  | size of the slider                                                                                        | string          | large / default /small | default |
```
Comment (JeremyWuuuuu) [inline docs/en-US/component/slider.md:109]: I think you have missed a space after `default /`,

File: docs/en-US/component/slider.md [hunk 2]
```diff
@@ -97,7 +106,8 @@ slider/show-marks
...
+| input-size            | size of the input box, when set `size`, the default is the value of `size`                                | string          | large / default /small | default |
```
Comment (JeremyWuuuuu) [inline docs/en-US/component/slider.md:110]: Same on this.

File: docs/en-US/component/slider.md (hunk 1 above)
Comment (kooriookami) [reply docs/en-US/component/slider.md:109]: Sure

**Review by jw-foss — APPROVED** [review]
LGTM, /cc @HerringtonDarkholme

**Comment by kooriookami** [comment]
#5103

**Pull request merged**